packages/rich-text-editor/test/a11y.common.js

```diff
@@ -94,6 +94,15 @@ describe('accessibility', () => {
   });
 
   describe('keyboard navigation', () => {
+    beforeEach(async () => {
+      rte = fixtureSync('<vaadin-rich-text-editor></vaadin-rich-text-editor>');
+      await nextRender();
+      editor = rte._editor;
+      buttons = Array.from(rte.shadowRoot.querySelectorAll(`[part=toolbar] button`));
+      content = rte.shadowRoot.querySelector('[contenteditable]');
+      announcer = rte.shadowRoot.querySelector('[aria-live=polite]');
```

Comment:
Seems like `announcer` isn't used in this suite, can be removed.

Reply:
Done.